chore/find_javadoc_regression.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import re
+import sys
+from collections import Counter
+from pathlib import Path
+from typing import List
+
+
+def filter_relevant_lines(lines: List[str]) -> List[str]:
+    """Tries to filter the checkstyle output to only include lines with violations."""
+    return [line for line in lines if re.search(r":\d+:", line) and "ERROR" in line]
+
+
+def strip_line_number(line: str) -> str:
+    return re.sub(r".java:\d+(:\d+)?:", ".java", line)
+
+
+def find_added_violation_lines(reference: 'Counter[str]', other: 'Counter[str]') -> 'Counter[str]':
+    """
+    Tries to find out which violation lines are new in "other" and which
+    were already present in the "reference". These lines are added violations the
+    contributor should fix.
+    """
+    # Lines that are only in the reference aren't critical, as they were
+    # apparently fixed in "other"
+    return other.copy() - reference
```

Comment:
Very minor note here: there's no need to copy `other`, as the subtraction is not in-place (i.e. does not mutate `other`).

```suggestion
    return other - reference
```

Reply:
I actually looked that up in the docs because I wasn't sure and didn't want to needlessly copy things aroound. The `subtract` method *does* modify the counter but the binary operator doesn't... It makes sense now that I think about it, I'll adjust it.

Thank you :)